source/api/v1/auth/redis.ts

```diff
@@ -0,0 +1,36 @@
+import { createClient } from "async-redis";
```

Comment:
I have issue with redis connection. Could you check in your environment? It doesn't work with redis but work with async-redis. But still I get error in createClient: Module '"async-redis"' has no exported member 'createClient'.

Reply:
Fixed

Reply:
👍